tests/databrowser.js

```diff
@@ -187,14 +187,14 @@ define([
                     editor = label.ckeditor().editor,
                     widget = util.getWidget('itext-en-label');
                 widget.input.promise.then(function () { 
-                    editor.on('change', function() {
+                    editor.on('change', _.debounce(function() {
```

Comment:
Adding the placeholder text means that the call to `insertHtml` now fires two change events (guessing one when the placeholder is removed and then one when the new text is added), and the asserts won't pass until the latter one. I don't love adding this timing, but it seemed less fragile than making the test expect exactly two change events...ckeditor docs say: `This event may thus in some cases be fired when no changes happen or may even get fired twice.` which makes me wary of relying on how many times it's fired.